Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Maya/constants.py

```diff
@@ -8,9 +8,330 @@
 
-"""DccScriptingInterface (DCCsi)
-This module contains constants for the O3DE Maya DCCsi interface
+This module contains default values for commony used constants & strings.
+We can make an update here easily that is propogated elsewhere.
 """
+# -------------------------------------------------------------------------
+# built-ins
+import sys
+import os
+import site
+import timeit
+import inspect
+from os.path import expanduser
+from pathlib import Path
+import logging as _logging
+# -------------------------------------------------------------------------
+
+
+# -------------------------------------------------------------------------
+_START = timeit.default_timer() # start tracking
+
+# global scope
+_MODULENAME = 'Tools.DCC.Maya.constants'
+_LOGGER = _logging.getLogger(_MODULENAME)
+_LOGGER.debug('Initializing: {}.'.format({_MODULENAME}))
+# -------------------------------------------------------------------------
+
+
+# -------------------------------------------------------------------------
+# Maya is frozen
+#_MODULE_PATH = Path(__file__)
+# https://tinyurl.com/y49t3zzn
+# module path when frozen
+_MODULE_PATH = Path(os.path.abspath(inspect.getfile(inspect.currentframe())))
+_LOGGER.debug('_MODULE_PATH: {}'.format(_MODULE_PATH))
+
+_PATH_DCCSI_TOOLS_MAYA = Path(_MODULE_PATH.parent)
+_PATH_DCCSI_TOOLS_MAYA = Path(os.getenv('PATH_DCCSI_TOOLS_MAYA',
+                                        _PATH_DCCSI_TOOLS_MAYA.as_posix()))
+
+_PATH_DCCSI_TOOLS_DCC = Path(_PATH_DCCSI_TOOLS_MAYA.parent)
+_PATH_DCCSI_TOOLS_DCC = Path(os.getenv('PATH_DCCSI_TOOLS_DCC',
+                                       _PATH_DCCSI_TOOLS_DCC.as_posix()))
+
+_PATH_DCCSI_TOOLS = Path(_PATH_DCCSI_TOOLS_DCC.parent)
+_PATH_DCCSI_TOOLS = Path(os.getenv('PATH_DCCSI_TOOLS',
+                                   _PATH_DCCSI_TOOLS.as_posix()))
+
+# we need to set up basic access to the DCCsi
+_PATH_DCCSIG = Path(_PATH_DCCSI_TOOLS.parent)
+_PATH_DCCSIG = Path(os.getenv('PATH_DCCSIG', _PATH_DCCSIG.as_posix()))
+site.addsitedir(_PATH_DCCSIG.as_posix())
+
+_LOGGER.debug('_PATH_DCCSIG: {}'.format(_PATH_DCCSIG.as_posix()))
+
+# this is the shared default requirements.txt file to install for python 3.6.x+
+DCCSI_PYTHON_REQUIREMENTS = Path(_PATH_DCCSIG, 'requirements.txt').as_posix()
+# if using maya 2020 or less with py2.7 override with and use the one here:
+# "DccScriptingInterface\Tools\DCC\Maya\requirements.txt"
+
+# now we have azpy api access
+from azpy.env_bool import env_bool
+from azpy.constants import ENVAR_DCCSI_GDEBUG
+from azpy.constants import ENVAR_DCCSI_DEV_MODE
+from azpy.constants import ENVAR_DCCSI_LOGLEVEL
+from azpy.constants import ENVAR_DCCSI_GDEBUGGER
+from azpy.constants import FRMT_LOG_LONG
+# -------------------------------------------------------------------------
+
+
+# -------------------------------------------------------------------------
+from azpy.constants import * # but here are the specific ones we are gonna use
```

Comment:
Is this import needed?

Reply:
I will remove next PR when I have more time to make sure nothing breaks